lib/iob/index.js

```diff
@@ -15,6 +15,10 @@ function generate (inputs) {
     };
 
     var iobArray = [];
+    console.error(inputs.clock);
+    if (! /(Z|[+-][0-2][0-9]:[03]0)+/.test(inputs.clock) ) {
```

Comment:
I'd be inclined to make that regex just a little more general, maybe - possible there are some non-half-hour timezones?  Could just do something like \d\d:\d\d (though I don't know if JS uses that syntax for regex digits).  Most likely fine as-is, though.